frontend/apps/reader/modules/readertypography.lua

```diff
@@ -41,6 +41,7 @@ local HYPH_DICT_NAME_TO_LANG_NAME_TAG = {
     ["Russian_EnGB.pattern"] = { _("Russian + English (UK)"), "ru-GB" },
     ["Russian_EnUS.pattern"] = { _("Russian + English (US)"), "ru-US" },
     ["Russian.pattern"]      = { _("Russian"),      "ru" },
+    ["Serbian.pattern"]      = { _("Serbian"),      "sr" },
```

Comment:
We don't have any Serbian.pattern in https://github.com/koreader/crengine/tree/master/cr3gui/data/hyph/ 
The fact you added it here is may be a non-issue (it's for migration) - but I think then you could just let it out.

Reply:
Thanks, now I'm confused by this PR.